.github/workflows/reusable-build-test-release.yml

```diff
@@ -321,7 +321,7 @@ jobs:
           path: build/package/
 
       - name: VirusTotal Scan
-        uses: crazy-max/ghaction-virustotal@v3
+        uses: crazy-max/ghaction-virustotal@master
```

Comment:
@xjin-splunk can you please ask for release on https://github.com/crazy-max/ghaction-virustotal repo? It is bad practice to use a branch instead of tag/release. We can face many new problems which will be potentially in master